src/containers/Profile.vue

```diff
@@ -119,16 +113,19 @@ export default defineComponent({
       }
       return false;
     },
+    // FIXME: the requirement graph is not generated for the chosen major / minor / grad
```

Comment:
This wasn't super clear. Essentially what I mean here is that if you click to pick a new major (or college or minor or grad), we don't fetch from the cloud function to get the requirements for that major. Doing so would take say 0.5 seconds and so the warning would flash for 0.5 seconds, as the current requirement json has no information about whatever new choice you made and would hence think it is invalid.